modules/manually-creating-alibaba-ram-user.adoc

```diff
@@ -0,0 +1,246 @@
+access_key_secret = wYx56mszAN4Uunfh            # Secret
+----
+<1> Add your AccessKeyID and AccessKeySecret here. 
+
+* The RAM user must have the `AdministratorAccess` policy to ensure that the account has sufficient permission to create the {product-title} cluster. This policy grants permissions to manage all Alibaba Cloud resources.
++
+When you attach the `AdministratorAccess` policy to a RAM user, you grant that user full access to all Alibaba Cloud services and resources. If you do not want to create a user with full access, create a custom policy with the following actions that you can add to your RAM user for installation. These actions are sufficient to install {product-title}.
++
+[TIP]
+====
+You can copy and paste the following JSON code into the Alibaba Cloud console to create a custom poicy. For information on creating custom policies, see link:https://www.alibabacloud.com/help/en/doc-detail/93733.html[Create a custom policy] in the Alibaba Cloud documentation.
+====
++
+.Example custom policy JSON file
+[%collapsible]
+====
+[source,json]
+----
+{
+  "Version": "1",
+  "Statement": [
+    {
+      "Action": [
+        "tag:ListTagResources",
+        "tag:UntagResources"
+      ],
+      "Resource": "*",
+      "Effect": "Allow"
+    },
+    {
+      "Action": [
+        "vpc:DescribeVpcs",
+        "vpc:DeleteVpc",
+        "vpc:DescribeVSwitches",
+        "vpc:DeleteVSwitch",
+        "vpc:DescribeEipAddresses",
+        "vpc:DescribeNatGateways",
+        "vpc:ReleaseEipAddress",
+        "vpc:DeleteNatGateway",
+        "vpc:DescribeSnatTableEntries",
+        "vpc:CreateSnatEntry",
+        "vpc:AssociateEipAddress",
+        "vpc:ListTagResources",
+        "vpc:TagResources",
+        "vpc:DescribeVSwitchAttributes",
+        "vpc:CreateVSwitch",
+        "vpc:CreateNatGateway",
+        "vpc:DescribeRouteTableList",
+        "vpc:CreateVpc",
+        "vpc:AllocateEipAddress",
+        "vpc:ListEnhanhcedNatGatewayAvailableZones"
+      ],
+      "Resource": "*",
+      "Effect": "Allow"
+    },
+    {
+      "Action": [
+        "ecs:ModifyInstanceAttribute",
+        "ecs:DescribeSecurityGroups",
+        "ecs:DeleteSecurityGroup",
+        "ecs:DescribeSecurityGroupReferences",
+        "ecs:DescribeSecurityGroupAttribute",
+        "ecs:RevokeSecurityGroup",
+        "ecs:DescribeInstances",
+        "ecs:DeleteInstances",
+        "ecs:DescribeNetworkInterfaces",
+        "ecs:DescribeInstanceRamRole",
+        "ecs:DescribeUserData",
+        "ecs:DescribeDisks",
+        "ecs:ListTagResources",
+        "ecs:AuthorizeSecurityGroup",
+        "ecs:RunInstances",
+        "ecs:TagResources",
+        "ecs:ModifySecurityGroupPolicy",
+        "ecs:CreateSecurityGroup",
+        "ecs:DescribeAvailableResource",
+        "ecs:DescribeRegions",
+        "ecs:AttachInstanceRamRole"
+      ],
+      "Resource": "*",
+      "Effect": "Allow"
+    },
+    {
+      "Action": [
```

Comment:
One more permission, "pvtz:DeleteZoneRecord", needs to be added, per [Slack discussion](https://coreos.slack.com/archives/C01PES793LL/p1645685511922099?thread_ts=1644336131.581319&cid=C01PES793LL). Thanks!